timber/src/test/java/timber/log/TimberTest.java

```diff
@@ -209,6 +210,78 @@
     assertExceptionLogged("OMFG!", "java.lang.NullPointerException");
   }
 
+  @Test public void shortcutExceptionVerbose() {
```

Comment:
I would prefer to see all log levels put in a single test for each functionality. This way there's only two new tests. Also let's call these "exception only" instead of "shortcut exception".